bobtemplates/odoo/hooks.py

```diff
@@ -86,6 +87,25 @@ def _rm_suffix(suffix, configurator, path):
     os.rename(path, path[: -len(suffix)])
 
 
+def _get_oca_readme_fragments(configurator, addon_name):
+    url = (
```

Comment:
Maybe this as global variable to be quite visible?

Reply:
Sure bro

Reply:
Done !